ui/src/app/base/components/Meter/Meter.tsx

```diff
@@ -19,18 +52,41 @@ const Meter = ({
   labelsClassName,
   max,
   overColor = "#F99B11",
+  segmented = false,
+  separatorColor = "#F7F7F7",
   small = false,
-}) => {
+}: Props): JSX.Element => {
+  const el = useRef(null);
   const hasLabels = data.some((datum) => datum.label);
   const valueSum = data.reduce((sum, datum) => sum + datum.value, 0);
   const maximum = max || valueSum;
   const widths = data.map((datum) => (datum.value / maximum) * 100);
+  const [segmentSize, setSegmentSize] = useState(0);
+
+  useEffect(() => {
+    if (segmented) {
+      updateSegmentSize(el, maximum, setSegmentSize);
+    }
+  }, [maximum, segmented]);
```

Comment:
Is it likely segmented with change dynamically?

Reply:
Almost 0% chance probably, but the linter complains unless you include all the things that could possibly change.